hg-reset.py

```diff
@@ -117,19 +117,19 @@ def bail(parser,opt):
   stale,changed,unchanged=get_branches(ui,repo,heads_cache,marks_cache,mapping_cache,options.revision+1)
   good,bad=get_tags(ui,repo,marks_cache,mapping_cache,options.revision+1)
 
-  print "Possibly stale branches:"
-  map(lambda b: sys.stdout.write('\t%s\n' % b),stale.keys())
+  print("Possibly stale branches:")
+  list(map(lambda b: sys.stdout.write('\t%s\n' % b),list(stale.keys())))
 
-  print "Possibly stale tags:"
-  map(lambda b: sys.stdout.write('\t%s on %s (r%s)\n' % (b[0],b[1],b[3])),bad)
+  print("Possibly stale tags:")
+  list(map(lambda b: sys.stdout.write('\t%s on %s (r%s)\n' % (b[0],b[1],b[3])),bad))
```

Comment:
replace with for

Reply:
Do you mean following Syntax?

```suggestion
for items in bad:
    lambda b: sys.stdout.write('\t%s on %s (r%s)\n' % (b[0], b[1], b[3]))
```

Reply:
yes, I've meant this.